sdk/template/Azure.Template/stress/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsTestSupportProject>true</IsTestSupportProject>
```

Comment:
We shouldn't have to do this for actual stress projects. Can we add a convention based on the project name?

Reply:
Added `IsStressProject`, set by naming convention.